docs/set_up_tails.rst

```diff
@@ -38,7 +38,7 @@ links and then return to this page:
 -  `Download and verify the Tails
    .iso <https://tails.boum.org/download/index.en.html>`__
 -  `Install onto a USB
-   drive <https://tails.boum.org/doc/first_steps/installation/index.en.html>`__
+   drive <https://tails.boum.org/install/index.en.html>`__
```

Comment:
Looks good, but replacing this link would make both the "Download and verify" (in the line above) and the "Install onto a USB drive" links go to the same place (the first link redirects to the change you have in this diff). Can we replace the first link ("Download and verify the Tails .iso") with https://tails.boum.org/install/download/index.en.html ?